<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChunkEditor</class>
 <widget class="QWidget" name="ChunkEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2" rowspan="2">
    <widget class="QListWidget" name="tileList">
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QListWidget" name="chunkList">
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QFrame" name="viewerFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3"/>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>160</height>
      </size>
     </property>
     <property name="title">
      <string>Properties</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Tile Index</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="visualPlane">
        <item>
         <property name="text">
          <string>Low Plane</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>High Plane</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Visual Plane</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="tileIndex">
        <property name="maximum">
         <number>1023</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="flipY">
        <property name="text">
         <string>Flip Y</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="flipX">
        <property name="text">
         <string>Flip X</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="solidityA">
        <item>
         <property name="text">
          <string>Solid (All)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Solid (Top Only) (Grippable)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Solid (All Except Top)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Not Solid</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Solid (Top Only) (Not Grippable)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="solidityB">
        <item>
         <property name="text">
          <string>Solid (All)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Solid (Top Only) (Grippable)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Solid (All Except Top)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Not Solid</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Solid (Top Only) (Not Grippable)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Solidity (Path A)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Solidity (Path B)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="exportImg">
     <property name="text">
      <string>Export As Image</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
